scvi/models/log_likelihood.py

```diff
@@ -33,6 +33,8 @@ def compute_elbo(vae, posterior, **kwargs):
         )
         elbo += torch.sum(reconst_loss + kl_divergence).item()
     n_samples = len(posterior.indices)
+    kl_divergence_global = vae.compute_global_kl_divergence()
```

Comment:
1) I don't see why you need to call this here because shouldn't it be the term you returned by calling forward?
2) Doesn't the global KL need to be divided by sample size?

Reply:
so yeah, it looks like you should do the same normalization here as you do in the loss function.

Reply:
1. I thought it was ugly to use the global KL divergence returned by calling forward as it should always be the same and can be called only once, but doing so tackles Romain's previous point about unused `compute_global_kl_divergence` so this is the best idea.
2. We do divide it by sample size when we return `elbo / n_samples`